examples/results/forces.py

```diff
@@ -0,0 +1,60 @@
+import flow360 as fl
+from flow360.examples import OM6wing
+
+OM6wing.get_files()
+
+# submit mesh
+volume_mesh = fl.VolumeMesh.from_file(OM6wing.mesh_filename, name="OM6wing-mesh")
+volume_mesh = volume_mesh.submit()
+
+# submit case using json file
+params = fl.Flow360Params(OM6wing.case_json)
+case = volume_mesh.create_case("OM6wing-example", params)
```

Comment:
can we have a submit_case method to bypass create_case + submit?

Reply:
this is part of a larger design discussion. You can do: case.copy(), modify params, submit. **submit** is sending data to cloud. So I think making this behaviour patter for every operation will be more intuitive with users.